lib/single-new-eks-fargate-opensource-observability-pattern/grafanaoperatorsecretaddon.ts

```diff
@@ -0,0 +1,75 @@
+import 'source-map-support/register';
```

Comment:
Could we remove this file and reuse the file from OSS pattern.

Reply:
This is not taken care?

Reply:
done